loki/transformations/pragma_model.py

```diff
@@ -103,7 +103,7 @@ def pmap_unstructured_data(self, pragma, parameters, **kwargs):
             return Pragma(keyword='acc', content=f'enter data{content}')
         return self.default_retval()
 
-    def pmap_end_unstructured_data(self, pragma, parameters, **kwargs):
+    def pmap_exit_unstructured_data(self, pragma, parameters, **kwargs):
```

Comment:
Does this mean `end unstructured-data` will no longer work, i.e., breaking backwards-compatibility? I'm just wondering if we have used this somewhere that needs adapting in source.